### PR TITLE
feat: lighten dashboard-header's shadow

### PR DESCRIPTION
**Pull request by joaoviana**
Closes: N/A

### Description:

Lighten `pageheader`'s shadow.

before: 
image

after:
image

### CODE REVIEW

File: packages/frontend/src/components/common/Page/PageHeader.tsx [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-            radius="unset"
-            sx={{ zIndex: 1 }}
-        >
-            {children}
-        </Card>
-    );
-};
+type Props = { withShadow?: boolean; children: ReactNode };
+
+const PageHeader: FC<Props> = ({ withShadow, children }) => (
```
Comment (joaoviana) [inline packages/frontend/src/components/common/Page/PageHeader.tsx:8]: what do you think? 🤔 
@IrakliJani & @magnew

File: packages/frontend/src/components/common/Page/PageHeader.tsx [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+    <Card
+        component={Flex}
+        justify="flex-end"
+        align="center"
+        pos="relative"
+        h={PAGE_HEADER_HEIGHT}
+        px="lg"
+        py="md"
+        bg="white"
+        shadow={withShadow ? 'xs' : 'none'}
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/Page/PageHeader.tsx:18]: does none work? it must be doing a lookup to the theme config dictionary. it should probably be undefined

File: packages/frontend/src/components/common/Page/PageHeader.tsx (hunk 1 above)
Comment (IrakliJani) [reply packages/frontend/src/components/common/Page/PageHeader.tsx:8]: yeah, why not

Comment (IrakliJani) [reply packages/frontend/src/components/common/Page/PageHeader.tsx:8]: you don't need children type, FC comes with it

File: packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx [hunk 3]
```diff
@@ -188,7 +188,7 @@ const SavedChartsHeader: FC = () => {
                 </p>
             </Alert>
 
-            <PageHeader>
+            <PageHeader withShadow>
```
Comment (IrakliJani) [inline packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx:191]: so, only charts have headers but not dashboards right? is there any place other than that two where we use this?

**Comment by IrakliJani** [comment]
maybe it will be better with a light border, same border we have for view mode tiles? 🤔

**Comment by PriPatel** [comment]
There needs to be something here under the header , otherwise it blends into the tiles. 
Screenshot 2023-08-11 at 14 07 44

File: packages/frontend/src/components/common/Page/PageHeader.tsx (hunk 2 above)
Comment (joaoviana) [reply packages/frontend/src/components/common/Page/PageHeader.tsx]: it did bc it got ignored 😆

**Comment by joaoviana** [comment]
@IrakliJani & @PriPatel , I've made the changes to: https://github.com/lightdash/lightdash/pull/6710/commits/a341505dc51dafc8f60826c2c7534adaf3a33542

so i've reverted the `withShadow` prop, then just changed the `shadow` value. It looks good on both the chart and dashboard headers!

File: packages/frontend/src/components/common/Page/PageHeader.tsx [hunk 4]
```diff
@@ -3,24 +3,22 @@ import { FC } from 'react';
...
+    <Card
+        component={Flex}
+        justify="flex-end"
+        align="center"
+        pos="relative"
+        h={PAGE_HEADER_HEIGHT}
+        px="lg"
+        py="md"
+        bg="white"
+        shadow="0 0 0 1px #bec1c426"
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/Page/PageHeader.tsx:16]: This is a highly customized shadow. Can you leave a FIXME note to indicate that this shadow should be sourced from the theme config? As of now, we haven't modified the shadow configuration in the Mantine theme and are using the default Mantine settings

File: packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx (hunk 3 above)
Comment (joaoviana) [reply packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx]: This page header is only used in charts + dashboards!

File: packages/frontend/src/components/common/Page/PageHeader.tsx (hunk 4 above)
Comment (joaoviana) [reply packages/frontend/src/components/common/Page/PageHeader.tsx:16]: 100%, you're right... I'll add a FIXME yes 👍

**Review by IrakliJani — APPROVED** [review]

**Pull request merged**